backend/src/seeds/achievements.ts

```diff
@@ -0,0 +1,183 @@
+import { type Knex } from 'knex';
+
+const ActivityType = {
+    CYCLING: 'cycling',
+    RUNNING: 'running',
+    WALKING: 'walking',
+} as const;
+
+const Metric = {
+    KILOMETERS: 'km',
+    MINUTES: 'minutes',
+    STEPS: 'steps',
+    KILOCALORIES: 'kcal',
+} as const;
```

Comment:
looks like we can use type from shared, because these are seeds

Reply:
Thanks for checking. I will fix it.